<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimConnect Enforcer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
        <item>
         <widget class="LoggingConsole" name="loggingConsole">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="UIVector" name="vectorDisplay" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="SerialConsole" name="serialConsole"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>627</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalls">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSimConnect"/>
    <addaction name="actionConnectSerialPort"/>
    <addaction name="actionSaveCoefficients"/>
    <addaction name="actionLoadCoefficients"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuCalls"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSimConnect"/>
   <addaction name="actionCoefficientEditor"/>
   <addaction name="actionConnectSerialPort"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/feather/info.svg</normaloff>:/icons/resources/icons/feather/info.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About program</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/feather/info.svg</normaloff>:/icons/resources/icons/feather/info.svg</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionConnectSerialPort">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/feather/terminal.svg</normaloff>:/icons/resources/icons/feather/terminal.svg</iconset>
   </property>
   <property name="text">
    <string>C&amp;onnect</string>
   </property>
   <property name="toolTip">
    <string>Connect to serial port</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/feather/settings.svg</normaloff>:/icons/resources/icons/feather/settings.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/feather/refresh-ccw.svg</normaloff>:/icons/resources/icons/feather/refresh-ccw.svg</iconset>
   </property>
   <property name="text">
    <string>C&amp;lear</string>
   </property>
   <property name="toolTip">
    <string>Clear data</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/feather/log-out.svg</normaloff>:/icons/resources/icons/feather/log-out.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSimConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/feather/toggle-left.svg</normaloff>
     <normalon>:/icons/resources/icons/feather/toggle-right.svg</normalon>:/icons/resources/icons/feather/toggle-left.svg</iconset>
   </property>
   <property name="text">
    <string>SimConnect</string>
   </property>
   <property name="toolTip">
    <string>SimConnect</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionCoefficientEditor">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/feather/activity.svg</normaloff>:/icons/resources/icons/feather/activity.svg</iconset>
   </property>
   <property name="text">
    <string>CoefficientEditor</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionSaveCoefficients">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/feather/save.svg</normaloff>:/icons/resources/icons/feather/save.svg</iconset>
   </property>
   <property name="text">
    <string>SaveCoefficients</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionLoadCoefficients">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/feather/book-open.svg</normaloff>:/icons/resources/icons/feather/book-open.svg</iconset>
   </property>
   <property name="text">
    <string>LoadCoefficients</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>UIVector</class>
   <extends>QWidget</extends>
   <header>uivector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LoggingConsole</class>
   <extends>QPlainTextEdit</extends>
   <header>logging_console.h</header>
  </customwidget>
  <customwidget>
   <class>SerialConsole</class>
   <extends>QPlainTextEdit</extends>
   <header>serial_console.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
